smithy-rules-engine/src/main/java/software/amazon/smithy/rulesengine/validators/AuthSchemesValidator.java

```diff
@@ -139,6 +140,34 @@ private Stream<ValidationError> validateSigv4(
                     propertyIs(authScheme, SIGNING_REGION, Literal::asString));
         }
 
+        private Stream<ValidationError> hasAllKeys(
```

Comment:
formatting: continued lines, including method parameters and chained calls, should be indented two lines. This is done through the rest of this file for matching.